apps/web/next.config.js

```diff
@@ -376,6 +376,11 @@ const nextConfig = {
         destination: "/signup",
         permanent: true,
       },
+      {
+        source: "/auth",
+        destination: "/auth/login",
+        permanent: true,
```

Comment:
```suggestion
        permanent: false,
```

This is a temporary thing so we shouldn't make it permanent. Once a permanent redirect is done on browser, it is cached forever and can't be changed